tests/e2e/UpdateProfile.js

```diff
@@ -6,86 +6,85 @@ import UpdateProfile from './page-objects/UpdateProfile';
 });
 
-test('Direct upload using AJAX', async t => {
-  await logIn.login('test_user@test.com', 'password');
+test ('Direct upload using AJAX', async t => {
+  await logIn.login ('test_user@test.com', 'password');
   await t
-    .expect(notifications.messageType.success.innerText)
-    .eql(notifications.text.login);
+    .expect (notifications.messageType.success.innerText)
+    .eql (notifications.text.login);
   await t
-    .click(homePage.link.uploadFiles)
-    .setFilesToUpload(updateProfile.input.ajaxUpload, ['./uploads/hero.png']);
+    .click (homePage.link.uploadFiles)
+    .setFilesToUpload (updateProfile.input.ajaxUpload, ['./uploads/hero.png']);
   await t
-    .expect(updateProfile.files.name.innerText)
-    .eql('hero.png')
-    .expect(updateProfile.files.imgAWS.count)
-    .eql(1);
+    .expect (updateProfile.files.name.innerText)
+    .eql ('hero.png')
+    .expect (updateProfile.files.imgAWS.count)
+    .eql (1);
 });
 
-test('Uploading Files Directly to Amazon S3 and using uploaded file as an avatar', async t => {
-  await logIn.login('test_user@test.com', 'password');
+test ('Uploading Files Directly to Amazon S3 and using uploaded file as an avatar', async t => {
+  await logIn.login ('test_user@test.com', 'password');
   await t
-    .expect(notifications.messageType.success.innerText)
-    .eql(notifications.text.login)
-    .click(homePage.link.uploadFiles);
+    .expect (notifications.messageType.success.innerText)
+    .eql (notifications.text.login)
+    .click (homePage.link.uploadFiles);
   await t
-    .expect(
-      updateProfile.files.currentImgName.withText(
+    .expect (
+      updateProfile.files.currentImgName.withText (
         updateProfile.txt.currentAvatar
       ).exists
     )
-    .ok()
-    .expect(
-      updateProfile.files.currentImgName.withText(
+    .ok ()
+    .expect (
+      updateProfile.files.currentImgName.withText (
         updateProfile.txt.currentBanner
       ).exists
     )
-    .ok();
-  await t.setFilesToUpload(updateProfile.input.avatarUpload, [
+    .ok ();
+  await t.setFilesToUpload (updateProfile.input.avatarUpload, [
     './uploads/bug.png',
   ]);
-  await t.setFilesToUpload(updateProfile.input.banerUpload, [
+  await t.setFilesToUpload (updateProfile.input.banerUpload, [
     './uploads/bug.png',
   ]);
   await t
-    .expect(
-      updateProfile.files.currentImgName.withText(updateProfile.txt.newAvatar)
+    .expect (
+      updateProfile.files.currentImgName.withText (updateProfile.txt.newAvatar)
         .exists
     )
-    .ok()
-    .expect(
-      updateProfile.files.currentImgName.withText(updateProfile.txt.newBanner)
+    .ok ()
+    .expect (
+      updateProfile.files.currentImgName.withText (updateProfile.txt.newBanner)
         .exists
     )
-    .ok();
+    .ok ();
 
-  await t.expect(updateProfile.files.img.count).eql(2);
-  await t.expect(updateProfile.files.imgAWS.count).eql(2);
+  await t.expect (updateProfile.files.img.count).eql (2);
```

Comment:
all those spaces i think are syntax errors and it wont work

Reply:
fixed